src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js

```diff
@@ -809,9 +810,9 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
 
             for(var i = 0; i < $this.currentItems.length; i++) {
                 var stripedItem = $this.currentItems[i];
```

Comment:
I know you didn't make this variable but shouldn't "stripedItem" be "strippedItem" ?

Reply:
I think you are right, it should be "strippedItem".

Reply:
I did! My bad 😅